scripts/commerce.js

```diff
@@ -339,3 +339,12 @@ export function mapProductAcdl(product) {
     mainImageUrl: product?.images?.[0]?.url,
   };
 }
+
+export function getProductSku() {
+  const metadataSku = getMetadata('sku');
+  if (metadataSku) {
+    return metadataSku;
+  }
+
+  return getSkuFromUrl();
```

Comment:
This can be simpler, assuming `getMetadata` returns a falsy value like empty string or undefined or null:
```suggestion
  return getMetadata('sku') || getSkuFromUrl();
```